lib/sinon/restore-object.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+var getPropertyDescriptor = require("./util/core/get-property-descriptor");
+var walk = require("./util/core/walk");
+
+function restoreObject(object) {
+    walk(object || {}, function(prop, propOwner) {
+        if (
```

Comment:
You can reduce the indentation by using the return early pattern here.

This is very much a matter of style/preference, feel free to ignore.

Reply:
This feels better to me. If there was more than one line of code inside the `if`, I would be more compelled to switch this around. Also this was copied from one of the other files so it follows the same pattern.